docs/_docs/contributing/debugging/ide-debugging.md

```diff
@@ -147,6 +147,16 @@ The debugger should start and pause on your breakpoint.
 
 ![Start debugger](/images/contribution/start-debugger.jpg)
 
+If you are working on presentation compiler module, there is an easier way to start the debugger, without need of custom `launch.json`.
+It can be accomplished by running `Debug Test` code lens on exact test case.
+
+![Debug test via code lens](/images/contribution/debug-test-code-lens.jpg)
+
+The second way to run presentation compiler tests is to use test explorer tab which lists all available test cases.
+The debugger is started either by pressing debug button or by selecting `Debug Test` option from the menu.
+
+![Debug test via code lens](/images/contribution/debug-test-explorer.jpg)
+
```

Comment:
This should probably appear earlier, between `## Importing the project in VSCode using Metals` and `## Configuring the debugger`. Otherwise people will miss it and they will lose time trying to configure the `launch.json`.

Also it is not restricted to the presentation compiler tests. It can be used with any unit tests.